management-api-agent-common/src/main/java/com/datastax/mgmtapi/NodeOpsProvider.java

```diff
@@ -534,17 +534,18 @@ public Map<String, String> getReplication(@RpcParam(name = "keyspaceName") Strin
     return rows.one().getMap("replication", UTF8Type.instance, UTF8Type.instance);
   }
 
-  @Rpc(name = "getTables")
-  public List<String> getTables(@RpcParam(name = "keyspaceName") String keyspaceName) {
+  @Rpc(name = "getTables", multiRow = true)
+  public List<Table> getTables(@RpcParam(name = "keyspaceName") String keyspaceName) {
     String query =
         QueryBuilder.selectFrom("system_schema", "tables")
             .column("table_name")
+            .column("compaction")
             .where(Relation.column("keyspace_name").isEqualTo(QueryBuilder.literal(keyspaceName)))
             .asCql();
     UntypedResultSet rows = ShimLoader.instance.get().processQuery(query, ConsistencyLevel.ONE);
-    List<String> tables = new ArrayList<>();
+    List<Table> tables = new ArrayList<>();
     for (UntypedResultSet.Row row : rows) {
-      tables.add(row.getString("table_name"));
+      tables.add(new Table(row.getString("table_name"), row.getFrozenTextMap("compaction")));
```

Comment:
All that Reaper needs is the compaction class, but in case we need other fields later I think it's better to mimic the table's structure.